### ISSUE TITLE
Converter returns regex when passed UK

### ISSUE THREAD

**Issue by jmartinter** [konstantinstadler/country_converter#113]
Hi, I noticed that the library behaviour changed recently. When running
```
import country_converter as coco

cc = coco.CountryConverter()
cc.convert('UK', src="ISO2", to="ISO2", not_found="missing")
```
it returns `^uk$|^gb$` regex instead of the `GB` code returned previously. Is it intended?

**Comment by konstantinstadler** [comment]
Oh 🤦🏽  - I will fix that. Thanks for reporting

**Comment by konstantinstadler** [comment]
Fixed in 0.7.7

**Comment by jmartinter** [comment]
Thanks!
